tests/test_common.py

```diff
@@ -46,10 +46,10 @@
 df_right_pandas = pd.DataFrame({"c": [6, 12, -1], "d": [0, -4, 2]})
 df_right_lazy = pl.LazyFrame({"c": [6, 12, -1], "d": [0, -4, 2]})
 
-if os.environ.get("CI", None):
+if os.environ.get("CI", None):  # pragma: no cover
     try:
         import modin.pandas as mpd
-    except ImportError:  # pragma: no cover
+    except ImportError:
```

Comment:
For whatever reason, coverage was failing here and on `test_str.py` for the same reason